src/Elmah.Io.NLog/ElmahIoTarget.cs

```diff
@@ -63,11 +63,7 @@ protected override void Write(LogEventInfo logEvent)
 
         private string MachineName(LogEventInfo logEvent)
         {
-#if NET45
             return new MachineNameLayoutRenderer().Render(logEvent);
```

Comment:
Did you remove the #if because MachineNameLayoutRenderer is supported in netstandard1.6, or?